### PR TITLE
feat: consume Sage UI font from @sage/design-tokens package

### PR DESCRIPTION
**Pull request by Parsium**
### Proposed behaviour

- Consume `Sage UI` font files directly from `@sage/design-tokens` library. This should hopefully fix the flakey loading behaviour we are experiencing with Chromatic snapshots.
- Amend Getting Started Guide to reflect this change.

### Current behaviour

- `Sage UI` font files are loaded from an external CDN. Consuming apps also have to preload the font files in their project's HTML. 

### Checklist

- [x] Commits follow our style guide
- [x] Related issues linked in commit messages if required
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [x] Typescript `d.ts` file added or updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [ ] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required

### Testing instructions

Double check the fonts are loading correctly in Storybook and Chromatic.

### CODE REVIEW

File: src/style/fonts.css [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
 }
 
-@font-face{
+@font-face {
   font-family: "Sage UI";
-  src: url('https://fonts.sage.com/Sage_UI-Medium.woff2') format("woff2"), url('https://fonts.sage.com/Sage_UI-Medium.woff') format("woff");
-  font-weight: 700;
   font-style: normal;
-  font-display: swap;
+  font-weight: 400;
```
Comment (samtjo) [inline src/style/fonts.css:11]: The css file in the design-tokens package [here](https://github.com/Sage/design-tokens/blob/master/assets/fonts/sageui.css) uses font weights of 400, 500 and 700.

Why are we using different font weights to that package?

Comment (Parsium) [reply src/style/fonts.css:11]: Thanks for taking a look @samtjo. This is a misalignment between `design-tokens` and Carbon/legacy SOP apps, which @harpalsingh has flagged with DS and other product teams. Unfortunately if we were to use the 400/500/700 weights instead then our components won't look quite right. Therefore the plan is include this weight override for now in Carbon, then once we've updated the whole codebase to use typography tokens, we should be good to remove this override

**Review by jamime — COMMENTED** [review]
This should be a `feat` commit because we're changing `src` files.

**Review by jamime — APPROVED** [review]

**Review by samtjo — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Pull request merged**